.circleci/config.yml

```diff
@@ -154,7 +160,28 @@ jobs:
             - run:
                   name: Run Apex Tests
                   command: |
-                      node_modules/sfdx-cli/bin/run force:apex:test:run -u ciorg -c -r human
+                      mkdir -p ~/tests/apex
+                      node_modules/sfdx-cli/bin/run force:apex:test:run -u ciorg -c -r human -d ~/tests/apex -w 20
+            - persist_to_workspace:
+                  root: ~/
+                  paths:
+                      - .sfdx/*
```

Comment:
Samsies here. Apex results and maybe unpack the cli command '-r human', blah blah.

Reply:
Added